### DISCUSSION TITLE
How can you see organisations while searching in the web platform?

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#75340] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

While searching in the GitHub web platform how can you search for organisations?

**Comment by b8zeek** [comment, marked as answer]
Greetings friend. 👋🏼 

Try adding `type:org` as a prefix like this ➡️ `type:org THE_ORGANIZATION_YOU_ARE_SEARCHING_FOR`.

<img width="957" alt="Screenshot 2023-11-13 at 10 50 48" src="https://github.com/community/community/assets/24759563/ef72b783-9388-4039-88e0-239c22d4bc8c">

Cheers. 🍺 

🍀

**Reply by skanda890** [reply]
Oh, Thanks.
